src/shell/shell-primary-bar.tsx

```diff
@@ -328,11 +353,14 @@ const ShellPrimaryBar: FC<{ activeRoute: AppRoute }> = ({ activeRoute }) => {
 									view?.children.length > 0 &&
 									map(view?.children, (item) => (
 										<PrimaryBarElement
-											key={item.id}
-											onClick={(): void => history.push(`/${routes[item.id]}`)}
+											key={item?.id}
+											onClick={(): void => {
+												matomo.trackEvent('Primary Bar', `${item?.trackerLabel}`);
```

Comment:
@divyang-dolphin Please use constant for "primary bar"

Reply:
Resolved , Thanks